src/Modules/OrchardCore.Commerce/Views/OrderPart_Edit.cshtml

```diff
@@ -11,18 +12,47 @@
 
 @inject IMoneyService MoneyService
 @inject ISession Session
-@inject IPredefinedValuesProductAttributeService AttributesService
+@inject IPredefinedValuesProductAttributeService PredefinedAttributeService
+@inject IProductAttributeService AttributeService
 
 @{
```

Comment:
You should minimize business logic in cshtml. No static code analysis, error handling works differently, it's not a good time. I think almost the whole of this large code block should be in `OrderPartDisplayDriver.Edit()`